cmd/crio/config.go

```diff
@@ -107,7 +115,22 @@ runtime_untrusted_workload = "{{ .RuntimeUntrustedWorkload }}"
 #     privileged containers are by definition trusted and will always use the
 #     trusted container runtime. If default_container_trust is set to "trusted",
 #     CRI-O will use the trusted container runtime for all containers.
-default_workload_trust = "{{ .DefaultWorkloadTrust }}"
+#
+# DEPRECATED: The runtime handler should provide a key to the map of runtimes,
+#     avoiding the need to rely on the level of trust of the workload to choose
+#     an appropriate runtime.
+#     The support of this option will continue through versions 1.12 and 1.13.
+#     By version 1.14, this option will no longer exist.
+#default_workload_trust = "{{ .DefaultWorkloadTrust }}"
+
+  # The "crio.runtime.runtimes" table contains a list of OCI compatible
+  # runtimes.
+  # The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.
+  # The runtime to use is picked based on the runtime_handler provided by the CRI.
+  # If no runtime_handler is provided, the runtime will be picked based on the level
+  # of trust of the workload.
```

Comment:
I would leave out this line, since you're marking default_workload_trust as deprecated.

My interpretation of the new approach is that to implement the equivalent of `default_workload_trust=untrusted` without using deprecated options, you would set `runtime=<untrusted runtime>` (line 87), and then create another runtime named "trusted" (or any other name). Does that make sense?

Reply:
I agree with keeping the implementation fallback, I was just wondering if it should be omitted from the documentation. Then again, I think it's fine to leave it, and just remove that when the deprecated options are removed.